plasma_cash/child_chain/child_chain.py

```diff
@@ -67,6 +69,10 @@ def apply_transaction(self, transaction):
         return tx.hash
 
     def get_current_block(self):
+        # Query for deposit events everytime user calls `get_current_block`
+        events = self.deposit_filter.get_new_entries()
```

Comment:
I think there should be better way on getting a workaround on previous `watch` for web3. I don't really think this is safe and we would need to rely on remembering calling this function. (eg, If I call `get_block` then I would not be updated)

I did a quick research and found this [APScheduler](https://apscheduler.readthedocs.io/en/latest/userguide.html), which might be a good candidate. I did a quick look and prefer this over celery because if I remembered it right, celery needs to spin up its own process. With APScheduler, I think we can add job to `create_app`. How do you think?